### DISCUSSION TITLE
Field update of selected rows after export

### DISCUSSION THREAD

**Discussion by JohannesM007** [Budibase/budibase#9794] [Help] [answered]
Hi,
in my form table i have a status field and i want to change the value from "Open" to "Printed" after the selected rows has been exported with the "Export" button. What kind of action do i need to save the new status value in the table.
I tried "Update State" and "Save Row" but i guess i need the selected row id's for this update?
Thank you in advance ;-)
Johannes

![grafik](https://user-images.githubusercontent.com/125383022/221143256-aa342248-b8bb-49ea-93eb-2596ff8d6f09.png)

**Comment by avivkri** [comment, marked as answer]
Hello @JohannesM007,
The selected row ids are available through the binding `{{ <table-name>.Selected rows }}`. It will return a comma separated string if Handlebar is used or an array if Javascript is used. However, you may have to resort to Automations for bulk update with looping. Actions support update of only one row today.

**Comment by JohannesM007** [comment]
Hi @avivkri ,
thanks for the quick response and the hints, i will try the automation loop as mentioned.
Hope I can get this to work ;-)
